### ISSUE TITLE
Incompatible with Zotero 6.0.10?

### ISSUE THREAD

**Issue by zxdawn** [retorquere/zotero-better-bibtex#2225]
When I try to install the v6.7.17 into the newest Zotero, I got an incompatible error. But, v6.3.0 can be installed.

**Comment by retorquere** [comment]
BBT is tested every night on the latest version (on Linux), and I've just downloaded 6.7.17 and installed it on Zotero 6.0.10 on Mac, and both work. What platform are you on, and what error are you getting?

**Comment by zxdawn** [comment]
I'm using elementary OS :
```
Distributor ID:	Elementary
Description:	elementary OS 6.1 Jólnir
Release:	6.1
Codename:	jolnir

Linux version 5.11.0-40-generic (buildd@lgw01-amd64-010) (gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0, GNU ld (GNU Binutils for Ubuntu) 2.34) #44~20.04.2-Ubuntu SMP Tue Oct 26 18:07:44 UTC 2021
```
![image](https://user-images.githubusercontent.com/30388627/181247366-c186a5ec-df80-4b61-b827-c698834bdbae.png)

**Comment by retorquere** [comment]
How did you install Zotero?

**Comment by zxdawn** [comment]
Followed the steps [here](https://www.zotero.org/support/installation): Download the tarball, extract/copy the directory, and run.

**Comment by retorquere** [comment]
Do you know where I can download an image of elementary OS?

**Comment by retorquere** [comment]
NM, got it

**Comment by retorquere** [comment]
I can't install elementary in a VM, the install window paints what I assume must be the "install" or "continue" button outside the VM display.

**Comment by zxdawn** [comment]
Em ... Don't worry. I can continue to use the old one. Just curious about what's wrong with the newest version.

**Comment by retorquere** [comment]
I don't support old ones. The newest version is fine AFAICT, I've also just installed it on my Ubuntu machine without problems, so that makes 3 machines across two platforms that install without issue. Are you willing to try out the deb packages from https://github.com/retorquere/zotero-deb/ ?

**Comment by retorquere** [comment]
I also can't install elementary on virtualbox on linux, same problem with the screen size. Without access to a box with elementary, I cannot do anything. But it is simply not true that 6.7.17 is incompatible with zotero 6.0.10, and I'd love to fix your problem. I hate unresolved problems, and I will not be able to solve any problems you might encounter with 6.3.0.

**Comment by retorquere** [comment]
I'm not sure the apt error traces back to my zotero packages. The dependencies of the zotero packages are

```
Depends: gnupg, libatk1.0-0, libc6, libcairo-gobject2, libcairo2, libdbus-1-3, libdbus-glib-1-2, libfontconfig1, libfreetype6, libgdk-pixbuf2.0-0, libglib2.0-0, libgtk-3-0, libharfbuzz0b, libpango-1.0-0, libpangocairo-1.0-0, libpangoft2-1.0-0, libstdc++6, libx11-6, libx11-xcb1, libxcb-shm0, libxcb1, libxcomposite1, libxcursor1, libxdamage1, libxext6, libxfixes3, libxi6, libxrender1, libxt6
```

I also don't see apt installing zotero in your output. When I install on ubuntu, I see this:

```
$ sudo apt install zotero
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following package was automatically installed and is no longer required:
  crda
Use 'sudo apt autoremove' to remove it.
The following NEW packages will be installed:
  zotero
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 71,6 MB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 https://zotero.retorque.re/file/apt-package-archive ./ zotero 6.0.10 [71,6 MB]
Fetched 71,6 MB in 13s (5.337 kB/s)
Selecting previously unselected package zotero.
(Reading database ... 309864 files and directories currently installed.)
Preparing to unpack .../zotero_6.0.10_amd64.deb ...
Unpacking zotero (6.0.10) ...
Setting up zotero (6.0.10) ...
Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
Processing triggers for desktop-file-utils (0.26-1ubuntu3) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for shared-mime-info (2.1-2) ...
```

notice the `Get` -- that's not present in your output. 

What happens if you try to install anything else, eg `sudo apt install thunderbird`? I'm curious whether you would see the same "Unmet dependencies". If so, it looks like your install is b0rked.

**Comment by retorquere** [comment]
I finally managed to install elementary, and the debs install there without any problem whatsoever, and BBT installs without issue in those. I'm afraid you have broken your Elementary installation.

**Comment by heyrict** [comment]
I've also encountered this in Archlinux. Installing latest BBT `v6.7.17` gives me a "it may be incompatible with this version of Zotero" error, while `v6.7.16` works fine.

```sh
❯  uname -srv
Linux 5.18.11-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 12 Jul 2022 15:40:51 +0000
❯  zotero --version
Zotero Zotero 6.0.5, Copyright (c) 2006-2018 Contributors
```

**Comment by retorquere** [comment]
@heyrict that is a different issue, BBT 6.7.17 is indeed not compatible with Zotero 6.0.5, you will need to upgrade to at least Zotero 6.0.9.

**Comment by OctarineSourcerer** [comment]
A note for anyone else who had this issue; On Arch, I had this just now. I can't remember exactly which version of Zotero I had (I'd assumed up-to-date, I'd literally just updated everything), but after hitting "Find Updates" on BetterBibTeX I still had the same message. 
I ended up searching for the `zotero` package (I'm on Arch) - weird, I had it installed, but it wasn't available on the repositories. `zotero-bin` was, however, available on the AUR, and I installed it (uninstalled the current installation as part of the process). Plugins window wouldn't show, so I rebooted, and now everything's fine and dandy.

TL;DR: Installed `zotero-bin` from Arch repos and rebooted, fixed it for me. Still not sure which of the two (or both) did it, but there we go.

**Comment by retorquere** [comment]
@zxdawn I don't see what I can do here, short of logging in in on your system and fixing your apt state. Unless you have other suggestions I am going to have to close this issue.

**Comment by zxdawn** [comment]
@retorquere The `Get` step isn't shown because I have run that before.
The broken package is discussed on [stackoverflow](https://askubuntu.com/questions/1337203/how-to-use-apt-to-force-remove-corrupting-package), but no answer yet.

Anyway, I have switched to Mac OS and everything works well .... I'm gonna close this and if users have similar problem, they can check it.
